rpc/src/server/tezedge_context_remote.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) SimpleStaking and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+
+use std::sync::Arc;
+
+use crypto::hash::ContextHash;
+use failure::Fail;
+use tezos_new_context::{ContextError, ContextKeyOwned, ContextValue, StringTreeEntry};
+use tezos_wrapper::TezosApiConnectionPool;
+
+#[derive(Clone)]
+pub struct TezedgeContextRemote {
```

Comment:
@tizoc 
I would put this maybe to `tezos/wrapper` module, that we could use it maybe in shell, if needed

Reply:
Done and renamed to `*Client`, much better name.